persistence/src/main/java/cz/muni/fi/pa165/dao/BottleDAO.java

```diff
@@ -25,6 +27,14 @@
 
     List<Bottle> findByToxic(boolean toxic);
 
+    @Query("SELECT b FROM Bottle b " +
+            "INNER JOIN b.bottleType bt " +
+            "INNER JOIN bt.manufacturedBy m " +
+            "WHERE bt.id=b.bottleType " +
```

Comment:
Ja som to prepisal takto: @Query("SELECT b FROM Bottle b " +
            "INNER JOIN b.bottleType bt " +
            "INNER JOIN bt.manufacturedBy m " +
            "WHERE bt.manufacturedBy=:manufacturer " +
            "AND b.produced>=:date")
    List<Bottle> getAllBottlesFromManufacturerFromDate(
            @Param("manufacturer") Manufacturer manufacturer,
            @Param("date") Date date);

    @Query("SELECT b FROM Bottle b " +
            "INNER JOIN b.bottleType bt " +
            "INNER JOIN bt.manufacturedBy m " +
            "WHERE bt.manufacturedBy=:manufacturer")
    List<Bottle> getAllBottlesFromManufacturer(
            @Param("manufacturer") Manufacturer manufacturer);

A testy co si dopisal, mi normalne presli